### DISCUSSION TITLE
I am not able to get Cascadia Code's cursive working properly in Visual Studio Code. Please Help me in fixing It...

### DISCUSSION THREAD

**Discussion by BornAsSlayer** [microsoft/cascadia-code#768] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/a4b1e04d-6816-4d52-9097-894f971b32f6)

This is how my VSC settings look.
I used [v2404.23](https://github.com/microsoft/cascadia-code/releases/tag/v2404.23) this release and installed all non-static .tff fonts.

![image](https://github.com/user-attachments/assets/ba6d8365-7fdd-4b56-9147-e6bc05c4302a)

![image](https://github.com/user-attachments/assets/f65f8f5c-44c6-490a-8ba7-748f37ff5ec1)

this is my expected result...
![image](https://github.com/user-attachments/assets/0c2ef514-493b-4657-912d-12dd89951d9c)

**Comment by alexeyten** [comment, marked as answer]
You should use color theme that uses cursive (italic). Try [Night Owl](https://marketplace.visualstudio.com/items?itemName=sdras.night-owl).

**Reply by BornAsSlayer** [reply]
Oh my god.... Is that even a thing? But thank you so much it worked. <3

**Comment by Poshfroshy** [comment]
I'm having the same issue, i downloaded the cascadia font but dont know how to get i`t installed on my computer

**Reply by BornAsSlayer** [reply]
![image](https://github.com/user-attachments/assets/b8ae6e02-6138-4993-93b7-75ebe7d7c564)

That's all click to install/ install for all users.

There are also a lot of tutos in YT which teaches you how to do it too.

**Reply by BornAsSlayer** [reply]
This is a different font tho.... but the idea is same

**Reply by Poshfroshy** [reply]
I resolved it too

**Reply by Poshfroshy** [reply]
I used another theme with it and it worked
